docs/guides/bulk/index.md

```diff
@@ -0,0 +1,15 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Overview | How to Use Bulk
+title: Overview
+description: An overview of Cloud Element's Bulk Framework
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 1
+---
+
+# Bulk
+
+Cloud Elements provides the ability to upload and download data in bulk from an endpoint in a normalized way. Cloud Elements will leverage the provider bulk endpoints whenever that is available. When there is no bulk available from the provider, Cloud Elements will provide a pseudo bulk service for uploading and downloading data from the endpoint. For uploads, we will accept a file and then create objects at the endpoint on a record by record basis. For downloads, we will execute a search API against the endpoint, and loop through all results until we have retrieved all the data. Cloud Elements will store these files (encrypted) in our platform for a maximum of three days. We’re currently rolling out a Bulk service on an Element by Element basis. Let us know if you need Bulk APIs for a given Element by contacting us at support@cloud-elements.com.
```

Comment:
Overall, avoid future tense. It usually adds words and makes things a little harder to read. I'm also going to recommend not capitalizing a lot of things that really aren't proper nouns, mostly for consistency, even though it won't be entirely consistent with how we do it now. So, wherever you use bulk or element in a sentence, make it sentence case.

Sentence 2:"Cloud Elements leverages the provider bulk endpoints whenever available." Shorter, clearer, and avoids future tense, which is harder to read.
Sentence 3: "...Cloud Elements ~~will~~  provides a ~~pseudo~~ bulk service for uploading and downloading data from the endpoint" unless it's important to know, otherwise it's a potentially confusing word.
Sentence 4: "For uploads, we ~~wil~~l accept a file and then create objects at the endpoint on a record-by-record basis." Record-by-record as a compound adjective (2 or more words that together modify a noun) needs a hyphen.
Sentence 5: "For downloads, we ~~will execute a~~ search ~~API~~ against the endpoint, and loop through all results until we ~~have~~ retrieve~~d~~ all the data." Execute is a good word if speaking only to a developer, which is a huge part, but not our only audience here, I'd get rid of it.
Sentence 6: "Cloud Elements ~~will~~ stores these files (encrypted) in our platform for a maximum of three days."
Sentence 7: Make a new paragraph here and hyphenate element-by-element